inlong-sort/sort-common/src/test/java/org/apache/inlong/sort/protocol/transformation/relation/InnerJoinNodeRelationTest.java

```diff
@@ -38,10 +38,15 @@
 import static org.junit.Assert.assertEquals;
 
 /**
- * Tests for{@link InnerJoinNodeRelationTest}
+ * Tests for {@link InnerJoinNodeRelationTest}
```

Comment:
`InnerJoinNodeRelationTest` should be InnerJoinNodeRelation

Reply:
done